src/bootstrap.js

```diff
@@ -1,21 +1,21 @@
-import 'polyfill';
-import 'asset/style/screen.scss';
 import 'modernizr';
-import 'settings';
 
 import Vue from 'vue';
-import filter from 'filter';
-import directive from 'directive';
-import component from 'component';
-import getRouter from 'router';
-import getStore from 'store';
-import startUp from 'control/startUp';
-import getLocaleConfig from 'config/localeConfig';
 import VueI18nManager from 'vue-i18n-manager';
 import { sync } from 'vuex-router-sync';
-import setupInjects from 'util/setupInjects';
-import localeLoader from 'util/localeLoader';
-import App from 'App';
+import './asset/style/screen.scss';
+import './polyfill';
+import directive from './directive';
+import component from './component';
+import getRouter from './router';
+import getStore from './store';
+import startUp from './control/startUp';
+import getLocaleConfig from './config/localeConfig';
+import setupInjects from './util/setupInjects';
+import localeLoader from './util/localeLoader';
+import App from './App';
+import filter from './filter';
+import './settings';
```

Comment:
@flut1 The settings need to be imported before any vue component so it's better and saver to place it on top. At least above getRouter and App. This is needed because otherwise we have sensible defaults in vue types a feature that me and Narie don't like. Disabling the sensible defaults will only have affect when it's set before using vue types.

Reply:
I'm not sure that the import order as you write it down is guaranteed to be the actual order modules are loaded and executed in. I'm pretty sure it isn't

Reply:
See: https://stackoverflow.com/questions/35551366/what-is-the-defined-execution-order-of-es6-imports

Reply:
It's not really about the order in which es6 imports work. It's about the order in which webpack adds those files to the bundle. Right now the settings file doesn't work. I can send you a simple setup that shows the difference.

Reply:
I'm not sure though if this does guarantee any kind of order in the webpack bundle. It might work in this case, but I'm not sure if writing imports in order works 100% of the time.

Reply:
The shared link talks about two specs, and that it could be async and that it's up to the implementation of the loader.

In this case, webpack transforms it to ES5 code and makes everything sync, where webpack keeps the order of imports.